k8s/nginx-configmap.yaml

```
@@ -0,0 +1,85 @@
+kind: ConfigMap
+metadata:
+  name: ssvirt-nginx-config
+  labels:
+    app: ssvirt-ui
+    component: nginx-proxy
+data:
+  nginx.conf: |
+    events {
+        worker_connections 1024;
+    }
+    
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)